lib/manager/gomod/artifacts.spec.ts

```diff
@@ -169,6 +170,46 @@ describe('.updateArtifacts()', () => {
     ).not.toBeNull();
     expect(execSnapshots).toMatchSnapshot();
   });
+  it('supports docker mode passthrough of GIT_CONFIG_COUNT', async () => {
```

Comment:
```suggestion

  it('supports docker mode passthrough of GIT_CONFIG_COUNT', async () => {
```
Add a newline before each new test